### DISCUSSION TITLE
enable embed / iframes for local network only?

### DISCUSSION THREAD

**Discussion by sevenlayercookie** [alexjustesen/speedtest-tracker#1045] [Q&A]
**loving the tracker first of all!**

I was hoping to embed the tracker in my webpage that is hosted on the same device as the tracker server (same device, but two webservers hosted on different ports).

I was hoping to embed an iframe of the tracker on my other webpage, but currently the tracker webserver is "refusing".

I'm new to webservers, so maybe my understanding is wrong, but if the port of my webserver is not forwarded publicly, shouldn't it be safe to have permissive CORS or CSP or whatever? how can I set the tracker server to be more permissive so that I may embed an iframe and access it within my local network?

**Comment by alexjustesen** [comment]
So "yes" is the short answer it would likely be fine but coding exceptions is something I'm trying to avoid to keep the code clean. 

I go into more detail here should you want to deep dive into it. 

https://github.com/alexjustesen/speedtest-tracker/issues/1024
